tools/idea-plugin/src/test/resources/lucide.css

```
@@ -0,0 +1,1708 @@
+@font-face {
+  font-family: "lucide";
+  src: url('lucide.eot?t=1771503639848'); /* IE9*/
```

Stylelint (17.3.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)